lblod/frontend-lokaal-mandatenbeheer
# BCSD Voorzitter Alert

## Description

A warning is shown if the voorzitter van BCSD is not a member of RMW or Vast Bureau

## How to test

- Add a voorzitter to BCSD
- A closeable warning should be shown
- Add that person as a member (lid) to RMW or Vast Bureau and the warning should disappear without needing a refresh
